docs/products/clickhouse/reference/supported-table-functions.rst

```diff
@@ -10,13 +10,17 @@ Table functions supported in Aiven for ClickHouse®
       SELECT *
       FROM deltaLake('s3://bucket/path/to/lake')
 
+Some of the table functions are explicitly disabled due to security concerns.
```

Comment:
```
.. note::

   Occasionally, you may find specific table functions disabled for security reasons.
```

Reply:
Frankly speaking I am not sure if this one reflects the Idea better.
But I am not a wording expert, I will trust you here)